app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Post List</h1>
\ No newline at end of file
+<div class="user">
+  <div class="user-avatar">
+    <%=image_tag("https://robohash.org/#{@user.id}.png", class: 'user--avatar', alt: 'user picture')%>
+  </div>
+  <div class="user-name">
+    <%=link_to @user.name, user_path(@user), class: 'name'%>
+    <%=link_to "Number of posts: #{@user.posts_counter}",          user_posts_path(@user), class: 'posts-num'%>
+  </div>
+</div>
+<%@posts.each do |post, idx|%>
+  <div class="user-post">
+    <h3><%=link_to post.title, user_post_path(@user, post) %></h3>
+    <p><%= post.text.slice(0...50) + '...' %></p>
+    <p class='count'>Comments <%=post.comments_counter%>, Likes <%=post.likes_counter%></p>
+    <div class="user-post-comments">
+      <h3>Comments</h3>
+      <%post.comments.each do |comment|%>
+        <p><%= comment.author.name %>: <%= comment.text %></p>
+      <%end%>
```

Comment:
Please be sure to use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page as it is part of the requirements 📝 You already have the proper method to achieve this 🤓 The most_recent_five_comments on post model file 💪